### PR TITLE
Add info chart to readme.

### PR DESCRIPTION
**Pull request by djosborne**
Add a chart to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ## Calico Network Policy for Kubernetes 
 
-This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  The controller uses the Kubernetes v1beta1 network policy API to configure Calico network policy.  The policy controller addon is deployed on top of Kubernetes as a set of replicated pods, and can also be installed as a static pod. 
+This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  
+
+![calico-policy-controller](calico-policy-controller.png)
+
+The controller uses the Kubernetes v1beta1 network policy API to configure Calico network policy.  The policy controller addon is deployed on top of Kubernetes as a set of replicated pods, and can also be installed as a static pod.
```
Comment (caseydavenport) [inline README.md]: I know this was there already, but the policy controller isn't actually a set of replicated pods - it's a single pod.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ## Calico Network Policy for Kubernetes 
 
-This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  The controller uses the Kubernetes v1beta1 network policy API to configure Calico network policy.  The policy controller addon is deployed on top of Kubernetes as a set of replicated pods, and can also be installed as a static pod. 
+This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  
+
+![calico-policy-controller](calico-policy-controller.png)
```
Comment (caseydavenport) [inline README.md]: The image isn't quite right.

The policy controller watches the following k8s resources:
- Pods
- Namespaces
- NetworkPolicies

What was the motivation for this README change? Do we really need a more in-depth page on "how it works", as discussed in https://github.com/projectcalico/k8s-policy/issues/30

**Comment by caseydavenport** [comment]
Closing this guy.

**Comment by gunjan5** [comment]
Why was this guy closed? I personally think a quick diagram explaining how it works would be very useful. +1 to adding this to a doc, but I also think a lot of people (especially developers) would look at README to quickly understand what it's about and a diagram like this would help. WDYT @caseydavenport @djosborne ?

**Pull request closed without merging**